docs/conf.py

```diff
@@ -185,8 +185,8 @@ def generate_kpi_categories(input_param_csv_file, input_cat_csv_file, output_rst
         cat_label = props.category
         df_param["in_category"] = df_param.category == props.category
 
-        parameter_per_cat = df_param.loc[df_param.in_category == True, "ref"].to_list()
-        parameter_names = df_param.loc[df_param.in_category == True, "label"].to_list()
+        parameter_per_cat = df_param.loc[df_param.in_category is True, "ref"].to_list()
+        parameter_names = df_param.loc[df_param.in_category is True, "label"].to_list()
```

Comment:
Pandas statements within `loc` are special, this is what causes the doc not to build ... I will reverse this change